src/layers/VivViewerLayer/VivViewerLayer.js

```diff
@@ -62,7 +62,10 @@ export default class VivViewerLayer extends CompositeLayer {
       getTileData,
       dtype,
       colorValues: paddedColorValues,
-      sliderValues: paddedSliderValues
+      sliderValues: paddedSliderValues,
+      updateTriggers: {
```

Comment:
https://github.com/uber/deck.gl/blob/3f67ea6dfd09a4d74122f93903cb6b819dd88d52/modules/geo-layers/src/tile-layer/tile-layer.js#L50 Can you add a comment that references this line?  I just want something mentioning that this triggers the re-creation of the cache/`Tileset2d` which in turn fires the `getTileData` function. I worry about them changing this condition so it would be good for future us to document this.